packages/airnode-examples/integrations/coingecko-post-processing/create-config.ts

```diff
@@ -181,6 +181,11 @@ const createConfig = async (generateExampleFile: boolean): Promise<Config> => ({
               environment: 'Node 14',
               timeoutMs: 5000,
               value: `
+                // Log out every coin data
+                input.forEach((coinData) => {
+                  console.log(\`[Post-processing snippet]: Received the following coin data: \\\${JSON.stringify(coinData, null, 2)}\`)
```

Comment:
Notice how the interpolation is escaped. Shall we mention this in the docs?

Reply:
Probably a good idea imo

Reply:
Very cool